recipe/meta.yaml

```
@@ -29,62 +29,81 @@ requirements:
   host:
     - python
     - pip
-    - cython >=0.29.32,<3
-    - numpy {{ numpy }}
-    - setuptools >=51.0.0
+    - setuptools >=61.0.0
     - wheel
+    - cython >=0.29.33,<3
+    # Using later version of numpy as it is available on all platforms, requirements for 2.0.3 are lower for pandas
+    - numpy 1.23.5
+    - versioneer
+    - tomli # [py<311]
   run:
     - python
-    # Dependencies
-    # see https://pandas.pydata.org/pandas-docs/version/1.5.3/getting_started/install.html#dependencies
+    # Base Dependencies
     - numpy >={{ numpy }},<2.0a0
-    - python-dateutil >=2.8.1
+    - python-dateutil >=2.8.2
     - pytz >=2020.1
+    - python-tzdata >=2022.1
     # Recommended dependencies to achieve large speedups
-    # see https://pandas.pydata.org/pandas-docs/version/1.5.3/getting_started/install.html#recommended-dependencies
-    - numexpr >=2.7.3
+    # see https://pandas.pydata.org/pandas-docs/version/2.0.3/getting_started/install.html#recommended-dependencies
+    - numexpr >=2.7.1
     - bottleneck >=1.3.2
   run_constrained:
     # Optional dependencies
-    # see https://pandas.pydata.org/pandas-docs/version/1.5.3/getting_started/install.html#optional-dependencies
-    - tzdata >=2022a
+    # see https://pandas.pydata.org/pandas-docs/version/2.0.3/getting_started/install.html#optional-dependencies
+    # and https://github.com/pandas-dev/pandas/blob/v2.0.3/pyproject.toml
+    # performance
+    - numba >=0.53.1
+    # plot
     - matplotlib-base >=3.3.2
+    # output_formatting
     - jinja2 >=3.0.0
     - tabulate >=0.8.9
+    # computation
     - scipy >=1.7.1
-    - numba >=0.53.1
-    - xarray >=0.19.0
-    - xlrd >=2.0.1
-    - xlwt >=1.3.0
-    - xlsxwriter >=1.4.3
+    - xarray >=0.21.0
+    # fss
+    - fsspec >=2021.7.0
+    # aws
+    - s3fs >=2021.08.0
+    # gcp
+    - gcsfs >=2021.7.0
+    - pandas-gbq >=0.15.0
+    # excel
+    - odfpy>=1.4.1
     - openpyxl >=3.0.7
     - pyxlsb >=1.0.8
-    - beautifulsoup4 >=4.9.3
-    - html5lib >=1.1
-    - lxml >=4.6.3
+    - xlrd >=2.0.1
+    - xlsxwriter >=1.4.3
+    # parquet, feather
+    - pyarrow >=7.0.0
+    # hdf5
+    - blosc >=1.21.0
+    - pytables >=3.6.1
+    # spss
+    - pyreadstat >=1.1.2
+    # DB
     - sqlalchemy >=1.4.16
     - psycopg2 >=2.8.6
     - pymysql >=1.0.2
-    - pytables >=3.6.1
-    - blosc >=1.21.0
-    - fastparquet >=0.4.0
-    - pyarrow >=1.0.1
-    - pyreadstat >=1.1.2
-    - fsspec >=2021.7.0
-    - gcsfs >=2021.7.0
-    - pandas-gbq >=0.15.0
-    - s3fs >=2021.08.0
+    # html, xml
+    - beautifulsoup4 >=4.9.3
+    - html5lib >=1.1
+    - lxml >=4.6.3
+    # compression
     - brotli >=0.7.0
     - python-snappy >=0.6.0
     - zstandard >=0.15.2
```

```suggestion
    - zstandard >=0.15.2
    # clipboard
    - pyqt >=5.15.1,<6
    - qtpy >=2.2.0
```
https://github.com/pandas-dev/pandas/blob/v2.0.3/pyproject.toml#L79